bigint/bigint.mbt

```diff
@@ -34,14 +34,14 @@ let karatsuba_threshold = 50
 
 // Useful bigints
 
-let zero : BigInt = from_int64(0L)
+let zero : T = from_int64(0L)
```

Comment:
shall we hold on changes to BigInt?

Reply:
I will remove the changes to BigInt in this pr.